content/en/docs/reference/access-authn-authz/rbac.md

```diff
@@ -462,12 +472,12 @@ Auto-reconciliation is enabled in Kubernetes version 1.6+ when the RBAC authoriz
 <tr>
 <td><b>system:basic-user</b></td>
 <td><b>system:authenticated</b> and <b>system:unauthenticated</b> groups</td>
-<td>Allows a user read-only access to basic information about themselves.</td>
+<td>Allows a user read-only access to basic information about themselves. As of 1.12 that role is [implemented here](https://github.com/kubernetes/kubernetes/blob/c3062bae218928b3ec94fa9bde84ead528604c75/plugin/pkg/auth/authorizer/rbac/bootstrappolicy/policy.go#L211).</td>
```

Comment:
What's the purpose of referencing the implementation? Do we intend to keep that link updated?

Reply:
It is a permalink to the implementation as of 1.12. I don't know how else to show people what "discovery endpoints" means besides this or copy/pasting the URLs. 

With my "ops" hat on I need to make a call whether to flip the anonymous auth flag and before I do that I would like to know which APIs are exposed.

I guess we can just say run kubectl get clusterorles

Reply:
removed links

Reply:
the kubectl command seems more evergreen. I don't feel that strongly, but immediately stale links raise an eyebrow